x/dispensation/types/querier.go

```diff
@@ -8,14 +8,15 @@ const (
 	QueryAllDistributions   = "distributions"
 	QueryRecordsByDistrName = "records_by_name"
 	QueryRecordsByRecipient = "records_by_recipient"
+	QueryClaimsByType       = "claims_by_type"
 )
 
 type QueryRecordsByDistributionName struct {
```

Comment:
I believe we should define types in proto now.

Reply:
This may already be done in the gRPC PR